diff --git a/node_modules/react-router5/types/BaseLink.d.ts b/node_modules/react-router5/types/BaseLink.d.ts
index 75eecd1..07c4bae 100644
--- a/node_modules/react-router5/types/BaseLink.d.ts
+++ b/node_modules/react-router5/types/BaseLink.d.ts
@@ -30,6 +30,7 @@ declare class BaseLink extends Component<BaseLinkProps, BaseLinkState> {
     callback(err: any, state: any): void
     clickHandler(evt: any): void
     render(): React.DetailedReactHTMLElement<
+        Pick<React.HTMLAttributes<HTMLElement>, 'inputMode'> &
         {
             href: any
             className: string
@@ -54,7 +55,6 @@ declare class BaseLink extends Component<BaseLinkProps, BaseLinkState> {
             style?: React.CSSProperties
             tabIndex?: number
             title?: string
-            inputMode?: string
             is?: string
             radioGroup?: string
             role?: string
